### PR TITLE
DOCS-3751 post reorg update services articles

### PR DESCRIPTION
**Pull request by clairewrtr**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md [hunk 1]
```diff
@@ -16,4 +19,4 @@ Log in to the TrueNAS web interface and go to **Services > Dynamic DNS**.
 The required values for these fields are provided by your DDNS solution.
 Start the DDNS service after choosing your **Provider** options and saving the settings.
 
-For more information on the configuration settings see  [**Dynamic DNS** screen]({{< relref "/CORE/UIReference/Services/DynamicDNSScreen.md" >}}).
\ No newline at end of file
+{{< taglist tag="coreddns" limit="10" title="Related Articles" >}}
```
Comment (micjohnson777) [inline content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md:22]: it is my understanding that using the title="Related xyz Articles" tag is only necessary if you are using the tag a second time to cover other related articles where you want to say essentially what the tag topic is. For example, the article is on DDNS but you want to show Related IP Articles and you use the coreip tag.

File: content/CORE/CORETutorials/Services/ConfigureUPS.md [hunk 2]
```diff
@@ -38,4 +41,4 @@ The secondary computers are powered by the UPS, but receive UPS status data from
 See the [NUT User Manual](https://networkupstools.org/docs/user-manual.chunked/index.html) and [NUT User Manual Pages](https://networkupstools.org/docs/man/index.html#User_man).
 {{< /expand >}}
 
-See [UPS screen]({{< relref "/CORE/UIReference/Services/UPSScreen.md" >}}) for more information on setting up a UPS.
\ No newline at end of file
+{{< taglist tag="coreups" limit="10" title="Related UPS Articles" >}}
```
Comment (micjohnson777) [inline content/CORE/CORETutorials/Services/ConfigureUPS.md:44]: Same comment as above regarding adding the title="Related xyz Articles" only when adding a second shortcode for a different tag. The "Related Articles" heading happens automatically with the basic shortcode without the title="".

**Review by micjohnson777 — COMMENTED** [review]
Suggest combing through the article descriptions and the taglist shortcodes per the first couple of comments added. I'll review the substance changes again after you are able to update the descriptions and taglists.  :-)
Overall, a very good effort!!

File: content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md (hunk 1 above)
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md:22]: Removed taglist title parameter, updated image to 13.0 directory, changed description to be more standardized.

File: content/CORE/CORETutorials/Services/ConfigureUPS.md (hunk 2 above)
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfigureUPS.md:44]: Changed description: "This article provides information on configuring UPS service on your TrueNAS."

File: content/CORE/CORETutorials/Services/ConfigureUPS.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
 ---
 title: "Configuring UPS Service"
+description: "How to configure UPS service for your TrueNAS"
```
[the remark this answers is not in the document]
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfigureUPS.md:3]: Changed description: "This article provides information on configuring UPS service on your TrueNAS."

File: content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md [hunk 4]
```diff
@@ -1,7 +1,10 @@
 ---
 title: "Configuring Dynamic DNS"
+description: "Configuring Dynamic DNS"
```
Comment (micjohnson777) [inline content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md:3]: The description front matter should not just be the same as the title of the article. Maybe uses something like, "This article provides information on configuring dynamic DNS on your TrueNAS." or some variation. Try to summarize what information is in this article or what the reader will find in it.

File: content/CORE/CORETutorials/Services/ConfigureUPS.md (hunk 2 above)
Comment (micjohnson777) [reply content/CORE/CORETutorials/Services/ConfigureUPS.md:44]: for some reason the title parameter still shows up in line 44?

File: content/CORE/CORETutorials/Services/ConfiguringFTP.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Configuring FTP"
+description: "Configuring FTP on your TrueNAS"
```
Comment (micjohnson777) [inline content/CORE/CORETutorials/Services/ConfiguringFTP.md:3]: The description parameter is a little too close to the title, so maybe make it more of a complete sentence? "This article provides information on how to configure File Transfer Protocol (FTP) on your TrueNAS." or some variation. Can be "This article provides instructions on to to set up FTP using **Services > FTP**. There are a couple of different ways to be more descriptive but not to verbose in the short title.

File: content/CORE/CORETutorials/Services/ConfiguringLLDP.md [hunk 6]
```diff
@@ -1,7 +1,10 @@
 ---
 title: "Configuring LLDP"
+description: "Configuring Link Layer Discovery Protocol (LLDP)"
```
Comment (micjohnson777) [inline content/CORE/CORETutorials/Services/ConfiguringLLDP.md:3]: Description comment is still too close to the title. Try a complete sentence like, "This article provides instructions on configuring Link Layer Discovery Protocol (LLDP) on your TrueNAS." You can use describes how to configure or set up, or some variation on the theme.

File: content/CORE/CORETutorials/Services/ConfiguringLLDP.md [hunk 7]
```diff
@@ -16,4 +19,4 @@ Select **Interface Description** and enter a **Country Code**. The location of t
 
 Turn on the LLDP service on.
 
-See [LLDP Screen]({{< relref "/CORE/UIReference/Services/LLDPScreen.md" >}}) for more information on the settings.
+{{< taglist tag="corelldp" limit="10" title="Related Articles" >}}
```
Comment (micjohnson777) [inline content/CORE/CORETutorials/Services/ConfiguringLLDP.md:22]: the title="related articles" is not needed in this shortcode as it is there by default.
```suggestion
{{< taglist tag="corelldp" limit="10" >}}
```

File: content/CORE/CORETutorials/Services/ConfiguringOpenVPN.md [hunk 8]
```diff
@@ -1,7 +1,10 @@
 ---
 title: "Configuring OpenVPN"
+description: "How to configure OpenVPN"
```
Comment (micjohnson777) [inline content/CORE/CORETutorials/Services/ConfiguringOpenVPN.md:3]: Same comment as in preceding articles. This is not a complete sentence and it is too close to the title text.

File: content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md:3]: "This article provides instructions on how to configure Dynamic Domain Name Service (DDNS) on your TrueNAS system."

**Review by micjohnson777 — COMMENTED** [review]
I'm seeing issues with the description parameters being incomplete sentences and text that is too close to the header text. Also the taglist shorcodes still have the title="" when you have only one taglist shortcode.
Some these articles need work in the body of the articles and parallel construction of headers. Also there are style issues as well but fewer and they were hiding in table descriptions. Suggest you review the articles in confluence and then take another stab at some of these how-to articles. We can improve the flow and even how they are written in general. :-)

File: content/CORE/CORETutorials/Services/ConfiguringFTP.md [hunk 9]
```diff
@@ -53,7 +54,5 @@ The default directory is the same as the user <file>/home</file> directory.
 After connecting, you can create directories and upload or download files.
 
 ![FilezillaFTPConnect](/images/CORE/FilezillaFTPConnect.png "Filezilla FTP Connection")
-
-See [FTP Screen]({{< relref "/CORE/UIReference/Services/FTPScreen.md" >}}) for more information on the **FTP** service screen. 
-
+
```
[the remark this answers is not in the document]
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfiguringFTP.md:57]: I've changed the initial level 2 header to FTP Connections, replaced the introductory sentences to be active voice, and replaced the initial image with a current CORE add dataset image.

File: content/CORE/CORETutorials/Services/ConfiguringLLDP.md (hunk 7 above)
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfiguringLLDP.md:22]: Deleted taglist title parameter for corelldp.

File: content/CORE/CORETutorials/Services/ConfiguringOpenVPN.md (hunk 8 above)
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfiguringOpenVPN.md:3]: Lines 12-16:
A virtual private network (VPN) is an extension of a private network over public resources. It allows remote clients on a public network to access a private network via a secure connection. TrueNAS provides OpenVPN as a system level service to provide VPN server or client functionality, and can act as a primary VPN server to allow remote clients access to data stored on the system using a single TCP or UDP port. TrueNAS can also integrate into a private network, even when the system is in a separate physical location or only has access to publicly visible networks.

Following level 2 header: Obtaining a Public Key Infrastructure (PKI)

Following level 2 header: Configuring OpenVPN: Process Overview

Following level 3 content already present for Configuring the OpenVPN Client and then OpenVPN Server.

Following level 2 headers for Connection Settings, Security Options and Service Activation.

File: content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md (hunk 1 above)
Comment (clairewrtr) [reply content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md:22]: These issues have been addressed in the corrections.

**Review by Mrt134 — COMMENTED** [review]
Completed review and found a few spots that I've updated locally.

**Review by Mrt134 — APPROVED** [review]
This is a good effort for style cleanup and the article link shortcodes are very helpful!

**Pull request merged**